catalogue/admin.py

```diff
@@ -255,6 +242,29 @@ def get_readonly_fields(self, request, obj=None):
         else:
             return ("approved",)
 
+    def changelist_view(self, request, extra_context=None):
+        list_display = (
+            "name",
+            "approved",
+            "published",
+            "allow_email",
+            "trl",
+            "description_short",
+            "get_created_by",
+            "created",
+            "lastmodified_at",
+        )
+        list_display_links = ("name",)
+        list_editable = ("published", "allow_email")
+        if is_admin_or_mod(request):
+            list_display = list_display[:1] + ("frontpage", ) +list_display[1:]
```

Comment:
insert space after plus

Reply:
maybe use a list instead of tuples? `list.insert()` would be nicer